can/test.js

```diff
@@ -2,6 +2,7 @@
 require("./map/map_test");
 require("./map/define-map-test");
 require("./map/observe-test");
+require("./map/observable-object-test");
```

Comment:
are `can-observe.Object.extend("Todo",{},{})` and `class Todo extends CanObserveObject` different enough internally to justify this extra test suite

Reply:
`can-observe` and `can-observable-object` are totally different.